### PR TITLE
Finish Tabs component

### PR DESCRIPTION
**Pull request by zzuu666**

### CODE REVIEW

File: components/tabs/index.en-US.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+| Property     | Description           | Type     | Default      |
+|--------------|-----------------------|----------|--------------|
+| activeKey    | Current TabPane's key| string   | - |
+| defaultActiveKey | Initial active TabPane's key, if `activeKey` is not set. | string | - |
+| onChange     | Callback executed when active tab is changed | Function | - |
+| onTabClick   | Callback executed when tab is clicked | Function | - |
+| onPrevClick  | Callback executed when prev button is clicked | Function | - |
+| onNextClick  | Callback executed when next button is clicked | Function | - |
+| tabBarExtraContent | Extra content in tab bar | React.ReactNode | - |
```
Comment (sky-admin) [inline components/tabs/index.en-US.md]: should remove react node?

File: components/tabs/tabs-content.vue [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      }
+      return style
+    }
+  },
+  methods: {
+    panes () {
+      return this.$children.filter(item => item.$options.name === 'pane')
+    }
+  },
+  updated () {
```
Comment (sky-admin) [inline components/tabs/tabs-content.vue]: never use?

File: components/tabs/tabs-content.vue [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      const prefixCls = this.prefixCls
+
+      return [
+        `${prefixCls}-content`,
+        this.animated ? `${prefixCls}-content-animated` : `${prefixCls}-content-no-animated`
+      ]
+    },
+    style () {
+      const marginLeft = this.activeIndex > -1 ? this.activeIndex * 100 : 0
+      const style = {
```
Comment (sky-admin) [inline components/tabs/tabs-content.vue]: can return directly

File: components/tabs/tabs-content.vue [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+<script>
+export default {
+  name: 'tabsContent',
+  props: {
+    prefixCls: {
+      type: String,
+      default: 'ant-tabs'
+    },
+    active: {
```
Comment (sky-admin) [inline components/tabs/tabs-content.vue]: never use？

File: components/tabs/tabs-tab.vue [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+import AtuIcon from '../icon'
+
+export default {
+  name: 'tabsTab',
+  props: {
+    prefixCls: {
+      type: String,
+      default: 'ant-tabs'
+    },
+    closable: {
```
Comment (sky-admin) [inline components/tabs/tabs-tab.vue]: mistake spell?

Comment (zzuu666) [reply components/tabs/tabs-tab.vue]: http://fanyi.baidu.com/#en/zh/closable

File: components/tabs/pane.vue [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+      type: [String, Number],
+      required: true
+    },
+    icon: {
+      type: String
+    },
+    tab: {
+      type: [String, Object]
+    },
+    closable: {
```
Comment (sky-admin) [inline components/tabs/pane.vue]: props order？

File: components/tabs/tabs-nav.vue [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+            .reduce((sum, value) => sum + value + gutter, 0)
+          : 0
+      })
+    },
+    getOffsetWH (node) {
+      return this.isVertical
+        ? node.offsetHeight
+        : node.offsetWidth
+    },
+    getOffsetLT (node) {
```
Comment (sky-admin) [inline components/tabs/tabs-nav.vue]: never use

Comment (zzuu666) [reply components/tabs/tabs-nav.vue]: For scrollToActive

**Pull request merged**